src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -714,10 +714,10 @@ public void setMetadata() {
 
     public static void updateSolidTransactions(Tangle tangle, Snapshot initialSnapshot, final Set<Hash> analyzedHashes)
```

Comment:
Change `final Set<Hash>` -> `LinkedHashSet<Hash>`
No need for final, and the order matters of course

Reply:
Changed To LinkedHashSet.